commands.go

```diff
@@ -277,6 +277,21 @@ func (cmd Command) Run(label string, cmdline ...string) error {
 	return nil
 }
 
+func (cmd Command) Run(label string, cmdline ...string) error {
+	w := newCommandWrapper(label, true)
+	defer w.flush()
+	return cmd.run(label, w, cmdline)
+}
+
+func (cmd Command) CheckExists(label string, cmdline ...string) error {
+	w := newCommandWrapper(label, false)
```

Comment:
is it worth being simple and just using `exec.LookPath` to check the executable exists somewhere rather than running the help commadn and checking the output ?

Reply:
If you'd prefer, I can make that change. Actually running the command seemed safest and simplest to me; it catches things like broken installs, missing libraries, and so forth. Just the presence of an executable doesn't mean you can use it.

Reply:
Fwiw i'm fine either way; I'm not sure trying to run things add much given well, so many things could potentially be broken. What if the tools are slightly different versions? what if a tool isn't bulid with the right options? etc etc. So i think going beyond, does it exist in the path has minimal extra value.. 

That said as things run in fakemachine that may potentially break programs, but that case seems both extremely rare and questionable if you'd detect that with just running `--help`. 

But again either way works for me as i don't think running help on a bunch of tools will have a negative impact either ;)